src/frontends/src/query_frontend.hpp

```diff
@@ -34,6 +34,16 @@ namespace query_frontend {
 const std::string LOGGING_TAG_QUERY_FRONTEND = "QUERYFRONTEND";
 const std::string GET_METRICS = "^/metrics$";
 
+const char* PREDICTION_RESPONSE_KEY_QUERY_ID = "query_id";
+const char* PREDICTION_RESPONSE_KEY_OUTPUT = "output";
+const char* PREDICTION_RESPONSE_KEY_USED_DEFAULT = "default";
+const char* PREDICTION_ERROR_RESPONSE_KEY_ERROR = "error";
+const char* PREDICTION_ERROR_RESPONSE_KEY_CAUSE = "cause";
```

Comment:
Our utility methods for adding key-value pairs to JSON objects expect `char*` keys. There is no implicit cast from `std::string` to `char*`, so I've defined the constants this way. Alternatively, I could define these with the `std::string` type and call `.data()` to get char* pointers, but this seems less readable.